test/functional/specs/Data Collector/C8118.js

```diff
@@ -49,7 +49,7 @@ const assertRequest = async request => {
   });
 };
 
-test("Test C8118: Verify link click sends a request to the collect endpoint when identity has been established, interact endpoint otherwise", async () => {
+test.skip("Test C8118: Verify link click sends a request to the collect endpoint when identity has been established, interact endpoint otherwise", async () => {
```

Comment:
I didn't mean to include this in this commit, since it's in https://github.com/adobe/alloy/pull/689. I'll just make sure I merge https://github.com/adobe/alloy/pull/689 first.